documentation/docs/strategies/cross-exchange-market-making.md

```diff
@@ -32,6 +32,16 @@ The end result: Hummingbot has sold the same asset at \$102 (❶) and purchased
 
 Initially, we assume that the maker exchange is an Ethereum-based decentralized exchange and that the taker exchange is Binance.
 
+### Adjusting Orders
+
+The maker price is calculated based on the price at which you can buy or sell on the taker exchange with a min_profitability added to it.
+
+For example if you can sell 1 ETH on taker exchange for 100 USDT on a volume weighted average price basis, then if min_profitability is 0.05, then maker price at which limit buy will be placed is 95 USDT.
+
+Now its possible that the top bid on the exchange is much lower, say 90 USDT. In this case if you enable adjust orders it will place at 90.01 (one price level above top bid).
+
+If you disable adjust orders, it will place order at 95 USDT instead of 90.01 USDT. 
+
```

Comment:
"adjust orders" seems too generic. Can we come up with a better term?

Reply:
what name do you suggest ?